src/DotNetLightning.Core/Channel/ChannelError.fs

```diff
@@ -373,7 +384,8 @@ module internal OpenChannelMsgValidation =
             check
                 ourChannelReserve (<) msg.DustLimitSatoshis
                 "Dust limit too high for our channel reserve. our channel reserve is: %A . received dust_limit is: %A"
-        Validation.ofResult(check1) *^> check2 *^> check3
+        //Validation.ofResult(check1) *^> check2 *^> check3
```

Comment:
commented code: please explain why it's left there (in a comment) or remove it

Reply:
oops I see this is part of the hacky commit

Reply:
thanks for adding the comment, but anyway I don't think this hunk should be merged in our fork (it's fine that your earlier versions of .Kiss include it, but this will be removed before we merge this PR)